src/tests/departments/team-lead.test.js

```diff
@@ -31,18 +30,21 @@ describe('department team lead', () => {
               code
               teamLead {
                 _id
-                fullName
+                firstName
+                lastName
                 email
               }
             }
           }
         }
       `
     })
-      .expect(res => {
+      .expect((res) => {
         // check if the user is assigned to the department
         expect(res.body).toHaveProperty('data.createOneDepartment.record')
-        expect(res.body.data.createOneDepartment.record.teamLead.fullName).toEqual(users[0].fullName)
+        expect(
+          res.body.data.createOneDepartment.record.teamLead.fullName
```

Comment:
You removed `fullName` but your assertion still checks for `fullName`

Reply:
`res.body.data.createOneDepartment.record.teamLead.fullName` will always be equal to `users[0].fullName` since they're both `null`, which makes your test pass - which is not the expected behavior